sites/partners/src/components/listings/PaperListingForm/sections/ApplicationTypes.tsx

```diff
@@ -248,63 +247,21 @@ const ApplicationTypes = ({ listing }: { listing: FormListing }) => {
               ]}
             />
           </GridCell>
-          {digitalApplicationChoice === YesNoAnswer.Yes && (
```

Comment:
nit: slight preference here to comment out as opposed to delete given that's how I've seen this done in other parts of the codebase